src/writing-policies/dsl.md

```diff
@@ -1,19 +0,0 @@
-> **WARNING:** not yet done, WIP
```

Comment:
Do not remove it, just leave it out of the `SUMMARY.md` file. I'll fix it with another PR